docs/src/content/docs/index.mdx

```diff
@@ -4,114 +4,137 @@ description: Terrateam documentation
+        <LinkCard title="Environments" href="/advanced-workflows/multiple-environments" />
 	</Card>
 	<Card title="Policy Enforcement" icon="document">
-		Integrate policy testing with OPA Conftest to ensure your infrastructure changes adhere to best practices and compliance requirements.
+		Integrate OPA Conftest to ensure compliance with best practices.
+        <LinkCard title="OPA Integration" href="/security-and-compliance/policy-enforcement-with-opa" />
 	</Card>
 </CardGrid>
 
-## Terraform the GitOps Way
+## How Terrateam Works
+
+Terrateam brings GitOps principles to Terraform workflows, making infrastructure changes visible, reviewable, and safe.
+
 <Steps>
-1. #### Create a Pull Request
+1. **Create a Pull Request with Terraform Changes**
 
+   When you create a pull request containing Terraform code changes, Terrateam automatically detects the modifications.
+   
    ![Terrateam pull request](../../assets/nibbler-pull-request.png)
 
-1. #### Terrateam will automatically run a Plan operation
+2. **Automatic Plan Operation**
 
+   Terrateam automatically runs `terraform plan` and comments the results directly in your pull request, allowing team members to review the proposed changes.
+   
    ![Terrateam plan](../../assets/nibbler-terrateam-plan.png)
 
-1. #### Apply your changes *before* or *after* you merge
+3. **Apply When Ready**
 
+   You can apply changes either before merging (to validate) or after merging (for production deployment), all through simple GitHub comments.
+   
    ![Terrateam Apply](../../assets/nibbler-terrateam-apply.png)
 
 </Steps>
+<LinkCard title="Learn more about workflows" href="/getting-started/pull-requests-and-triggers" />
 
-## Why Should I Use Terrateam?
-Terrateam provides a comprehensive set of features and best practices to streamline your infrastructure as code workflow, making it easier and safer to manage your Terraform projects.
+## Why Choose Terrateam?
 
 <CardGrid>
-  <Card title="Streamlined workflows" icon="laptop">
-    Terrateam takes care of the complex aspects of integrating Terraform with GitHub Actions, providing a seamless and efficient workflow out of the box. With Terrateam, you can focus on writing your Terraform code and let the tool handle the rest.
+  <Card title="Team Collaboration" icon="approve-check">
+    Review infrastructure changes with intelligent locking to prevent conflicts.
+    <LinkCard title="Collaboration Features" href="/advanced-workflows/locks-and-concurrency" />
   </Card>
-  <Card title="Get on the same page" icon="approve-check">
-    Terrateam enables effective collaboration among team members by providing a pull request-based workflow. Terrateam allows teams to review and discuss proposed changes before applying them. This ensures that everyone is on the same page and reduces the risk of errors and conflicts.
+  
+  <Card title="Enhanced Security" icon="seti:lock">
+    Fine-grained access controls, policy enforcement, and OIDC authentication.
```

Comment:
"Fine-grained" is repeated three times on the page. Try to use synonyms when possible.